ticdc/ticdc-changefeed-config.md

```diff
@@ -246,6 +246,14 @@ sasl-oauth-scopes = ["producer.kafka", "consumer.kafka"]
 sasl-oauth-grant-type = "client_credentials"
 # The audience in the Kafka SASL OAUTHBEARER authentication. The default value is empty. This parameter is optional when the OAUTHBEARER authentication is used.
 sasl-oauth-audience = "kafka"
+#The following configuration is only required when using Avro as the protocol and AWS Glue Schema Registry:
+#Please refer to the section "Using AWS Glue Schema Registry" in the document "Sync Data to Kafka" at the following link: https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-kafka#ticdc-集成-aws-glue-schema-registry
```

Comment:
I think this link should be changed to point to the English docs

Reply:
That's right, thank you!

Reply:
But it doesn't have English version in the English docs. https://docs.pingcap.com/tidb/dev/ticdc-sink-to-kafka#integrate-ticdc-with-kafka-connect-confluent-platform